### PR TITLE
feature/loop_bet

### PR DESCRIPTION
**Pull request by roygarrett**
Created betting loop for the play function. Testing was a little wonky due to the absence of the Pot class.

### CODE REVIEW

**Comment by roygarrett** [comment]
Hey @bennettapps @jonnyblankinship. You'll notice a lot of deletions. Do not be alarmed I just deleted a bunch of duplicate functions that were present in our dev_0 branch.

**Review by jonnyblankinship — COMMENTED** [review]
why does the bet input need to be in a while loop? Don't they just bet once?

File: test/test_bet_loop.py
Comment (jonnyblankinship) [inline test/test_bet_loop.py:1]: this test looks good

**Comment by roygarrett** [comment]
> why does the bet input need to be in a while loop? Don't they just bet once?

Hey Jonny! This bet input needs to be in a while loop so that the program will keep asking for bets until a valid bet is given!

File: test/test_bet_loop.py
Comment (roygarrett) [reply test/test_bet_loop.py:1]: Great! Thank you!

**Review by jonnyblankinship — APPROVED** [review]

File: test/test_bet_loop.py
Comment (bennettapps) [inline test/test_bet_loop.py:1]: you don't actually assert anything in your test so i don't think you need the import statement at the top?

**Review by bennettapps — COMMENTED** [review]
i don't think you need to import anything for your test since you never assert anything.

File: test/test_bet_loop.py
Comment (roygarrett) [reply test/test_bet_loop.py:1]: Hey Jesse! The test file imports the testing_base which gives the ability to execute all of the functions in the CardGames.py file!

**Review by bennettapps — APPROVED** [review]

**Pull request merged**